appveyor.yml

```diff
@@ -0,0 +1,6 @@
+version: 1.4.4-{build}
```

Comment:
Is this needed? Do we need to update this for every release or can we automate it?